Documentation/Books/Manual/DataModeling/Views/ViewMethods.md

```diff
@@ -115,16 +120,25 @@ Modify Properties
 
 `view.properties(view-property-modification)`
```

Comment:
signature must be changed to view.properties(view-property-modification, partialUpdate)

Reply:
Fixed.